canonical-websites/www.ubuntu.com
# Remove resource row heading margin top

## Done

- Remove resource row heading margin top
- Change some headings to make more semantic

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at:
  - [http://0.0.0.0:8001/internet-of-things/digital-signage](http://0.0.0.0:8001/internet-of-things/digital-signage)
  - [http://0.0.0.0:8001/internet-of-things/gateways](http://0.0.0.0:8001/internet-of-things/gateways)
  - [http://0.0.0.0:8001/internet-of-things/robotics](http://0.0.0.0:8001/internet-of-things/robotics)
  - [http://0.0.0.0:8001/desktop/1710](http://0.0.0.0:8001/desktop/1710)
  - [http://0.0.0.0:8001/server/maas](http://0.0.0.0:8001/server/maas)
  - [http://0.0.0.0:8001/openstack](http://0.0.0.0:8001/openstack)
  - [http://0.0.0.0:8001/cloud/managed-cloud](http://0.0.0.0:8001/cloud/managed-cloud)
  - [http://0.0.0.0:8001/kubernetes](http://0.0.0.0:8001/kubernetes)
  - [http://0.0.0.0:8001/kubernetes/managed](http://0.0.0.0:8001/kubernetes/managed)

- Make sure the resource type rows have reduced margin above heading links

### Old (bad)

![screen shot 2017-11-20 at 10 25 49](https://user-images.githubusercontent.com/2152/33013842-45590da6-cddd-11e7-9b6b-ae2098837ebf.png)

### New (good)

![screen shot 2017-11-20 at 10 26 08](https://user-images.githubusercontent.com/2152/33013854-4d1e7648-cddd-11e7-91af-edd9a7392a0e.png)

## Issue / Card

Fixes #2358
